multi/imxrt-multi/config.h

```diff
@@ -766,6 +766,63 @@
 #endif
 #endif
 
+#if !ISEMPTY(UART_CONSOLE)
+#if UART_CONSOLE <= 0
+#error "Invalid value for UART_CONSOLE"
+#endif
+#endif
+
+#if defined(UART_CONSOLE_USER)
+#if !ISEMPTY(UART_CONSOLE_USER)
+#if (UART_CONSOLE_USER <= 0)
+#error "Invalid value for UART_CONSOLE_USER"
+#endif
+#endif
+#else
+#define UART_CONSOLE_USER UART_CONSOLE
+#endif
+
+
+/* RTT */
+
+#ifndef RTT_CHANNEL0
+#define RTT_CHANNEL0 0
+#elif !ISBOOLEAN(RTT_CHANNEL0)
+#error "RTT_CHANNEL0 must have a value of 0, 1, or be undefined"
+#endif
+
+#ifndef RTT_CHANNEL1
+#define RTT_CHANNEL1 0
+#elif !ISBOOLEAN(RTT_CHANNEL1)
+#error "RTT_CHANNEL1 must have a value of 0, 1, or be undefined"
+#endif
+
+#ifndef RTT_CHANNEL0_BLOCKING
+#define RTT_CHANNEL0_BLOCKING 0
+#elif !ISBOOLEAN(RTT_CHANNEL0_BLOCKING)
+#error "RTT_CHANNEL0_BLOCKING must have a value of 0, 1, or be undefined"
+#endif
+
+#ifndef RTT_CHANNEL1_BLOCKING
+#define RTT_CHANNEL1_BLOCKING 0
+#elif !ISBOOLEAN(RTT_CHANNEL1_BLOCKING)
+#error "RTT_CHANNEL1_BLOCKING must have a value of 0, 1, or be undefined"
+#endif
+
+
+#ifndef RTT_CHANNEL_CONSOLE
```

Comment:
As above, I suggest using `RTT_CONSOLE_USER` to make board_config.h easier to read. There is no common `RTT_CONSOLE` define used, but maybe it's not needed for now, as an empty value is the default here

Reply:
Done